src/components/SortDropdown/types.ts

```diff
@@ -0,0 +1,11 @@
+export enum SortOptionsKeys {
+  APY = 'APY',
+  VOLUME = 'Volume',
+  MARKET_CAP = 'Market Cap',
+  PRICE_CHANGE = 'Price Change',
+}
```

Comment:
UpperCamelCase here too 🙏

Reply:
Done in [1ac600a](https://github.com/shapeshift/web/pull/8006/commits/1ac600aeb95da5729ef78865c837fd0bcec0f777)